### DISCUSSION TITLE
Support commit message commands syntax

### DISCUSSION THREAD

**Discussion by devinrhode2** [arxanas/git-branchless#877] [Ideas]
I've been using this syntax when creating certain git commits, which are purely the result of running some command:
```
exec `npm install prettier --save-dev
exec `npm run prettier --write .`
```

When these commands cause merge conflicts, a tool could simply discard the original `exec "..."` commit, and re-run the command on top of the other change.

This could make rebase scripts like this irrelevant: https://stackoverflow.com/questions/71812872/how-to-rebase-when-prettier-formatting-was-applied-to-whole-project

**Comment by arxanas** [comment]
This is the intended use-case for [`git test fix`](https://github.com/arxanas/git-branchless/wiki/Command:-git-test#fixing-formatting-and-linting-issues); does that work for you?
